manual/3-0_DecoderPro/Installing_JMRI.shtml

```diff
@@ -63,24 +62,21 @@
     </ul>
   </li>
 </ul>
-<p>&nbsp;</p>
 
 <hr>
 
-<div align=right>
+<div align="right">
 <p><a href="index.shtml#Installing">Back to Index</a></p>
 </div>
 
 
 This site is hosted by
 <br>
-<a href="http://sourceforge.net">
-<IMG src="images/sflogo.png" width="120" height="30" border="0" alt="SourceForge Logo" hspace="10" vspace="5" align="top"></a>
+  <a href="http://www.tagadab.com">
+    <img src="https://www.tagadab.com/sites/default/files/logo-tagadab-nostrap.png" height="28" width="103"
+         border="0" alt="Tagadab Logo"> </a>
 <br>
 </div>
 
-<!--#include virtual="/Footer.shtml" -->
-</div>
```

Comment:
Is this one perhaps missing a “div mBody”? We don’t have this structure well documented, but there are examples in /templates

Reply:
Especially these old _Manual_ pages don't follow the full CSS structure very well. Will try to clean them up a bit (anybody reading them?)

Reply:
Done